config/src/rpc_config.rs

```diff
@@ -226,7 +226,7 @@ pub struct IpcConfiguration {
     pub disable: bool,
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    #[clap(name = "ipc-apis", long, help = "rpc apiset to serve")]
+    #[clap(name = "ipc-apis", long, help = "ipc apiset to serve")]
```

Comment:
Perhaps it was correct and should not be modified.

Reply:
`rpc apiset` is right.